rpm/build/SPECS/jenkins.spec

```diff
@@ -76,30 +76,57 @@ rm -rf "%{buildroot}"
 /usr/sbin/useradd -g %{name} -s /bin/false -r -c "@@SUMMARY@@" \
 	-d "%{workdir}" %{name} &>/dev/null || :
 
+  # Used to decide later if we should perform a chown in case JENKINS_INSTALL_SKIP_CHOWN is false
+  # Check if a previous installation exists, if so check the JENKINS_HOME value and existing owners of work, log and cache dir, need to to this check
+  # here because the %files directive overwrites folder owners, I have not found a simple way to make the
+  # files directive to use JENKINS_USER as owner.
+  if [ -f "/etc/sysconfig/%{name}" ]; then
+      . /etc/sysconfig/%{name}
+      touch /tmp/cacheowner
+      ls -ld /var/cache/%{name} | awk 'NR==1 {print $3}' > /tmp/cacheowner
+      touch /tmp/logowner
+      ls -ld /var/log/%{name} | awk 'NR==1 {print $3}' >  /tmp/logowner
+      touch /tmp/workdirowner
+      ls -ld ${JENKINS_HOME:-%{workdir}}| awk 'NR==1 {print $3}'  > /tmp/workdirowner
+  fi
+
 %post
 /sbin/chkconfig --add %{name}
 
-# If we have an old hudson install, rename it to jenkins
-if test -d /var/lib/hudson; then
```

Comment:
it's not clear to me why this gets rid of the Hudson compatibility code. AFAIU this change is completely unrelated. I'd understand if this would allow implementing this PR, but that doesn't seem to be the case?